docs/make.jl

```diff
@@ -18,15 +18,19 @@ const EXAMPLES_DIR = joinpath(@__DIR__, "..", "examples")
 const OUTPUT_DIR   = joinpath(@__DIR__, "src/literated")
 
 to_be_literated = [
+    #"panantarctic_regional_simulation.jl",
```

Comment:
```suggestion
    "panantarctic_regional_simulation.jl",
```
the example is commented out